README.md

```diff
@@ -1,79 +1,99 @@
-- [Compilation](#️-compilation)
-- [Compilation Requirements](#-requirements)
-- [Compilation Steps](#-steps)
-- [Packaging](#-packaging)
-- [Built and Managed with](#️-built-and-managed-with)
-- [Contributing](#-contributing)
-- [Versioning](#-versioning)
-- [Authors](#-authors)
-- [License](#-license)
-- [Todos](#-todos)
-  </details>
+## 🎮 What is GDLauncher
+
+GDLauncher is a custom open source Minecraft launcher written from the ground up in electron/react. Its main goal is to make it easy and enjoyable to manage different Minecraft versions and install `forge/fabric`, bringing the playing and modding experience to the next level!
 
-<b>This project was previously developed in C#. That version is now outdated and not mantained but you can find the code [here](https://github.com/gorilla-devs/GDLauncher/tree/csharp_legacy_launcher)</b>
+## ⚡️ Comparison with Twitch Launcher
 
-GDLauncher is a custom open source Minecraft launcher written from the ground up in electron/react. Its main goal is to make it easy and enjoyable to manage different Minecraft versions and servers, bringing the playing and modding experience to the next level!
+This is an example of the time that GDLauncher takes to install a modpack in comparison to Twitch. Both tests are running at the same time over a 1Gbps network to ensure that the network doesn't impact the comparison.
+- GDLauncher: `0:52m`
+- Twitch Launcher: `2.25m`
+
+<p align="center">
+    <img width="800" height="auto" src="https://gdevs.io/comparison.gif" alt="GDLauncher" />
+</p>
 
 ## 🚀 Getting Started
 
-Below you will find everything you need to know about the launcher. If you want to download a stable release you can do it from our official website (https://gdevs.io). If you want the latest testing features you can clone the repo and compile it yourself.
+Below you will find everything you need to know about the launcher. If you want to download the latest stable release you can do it from our official website ([gdevs.io](https://gdevs.io)). If you want to test the possibly unstable features, you can clone the repo and compile it yourself.
 
 ## 🎮 Download
 
 To download the latest version, you can either click [here](https://github.com/gorilla-devs/GDLauncher/releases) and select the appropriate version for your operating system, or visit our [website](https://gdevs.io).
 
-## 🎨 Features (Working on)
+## 🎨 Features
 
-- Java Autorunner. You don't need to have java installed, a suitable version will be downloaded automatically.
-- Log console. Always know what's happening.
-- Easy installation. It's as easy as pie to install both the vanilla game and forge. No further action from the user is required.
-- Built-in autoupdater. You will never need to download a new version manually.
-- Vanilla, Forge, and Curse modpacks download and autoupdater.
-- Built-in manager for Minecraft servers.
-- Cloud sync of game saves. You will never lose your saves again!
+#### Our features:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5259002)